README.md

```diff
@@ -1 +1,89 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- Good job done so far, but for this project and your subsequent projects in Microverse, please make sure that you remove the microverse logo, optionally, you can choose to use your personal logo if you have any. This would help make your readme more professional. Please fix this 👍 👍